index.html

```diff
@@ -313,7 +313,13 @@ <h3 class="section-heading">Who's using LoopBack?</h3>
                 <div class="col-md-3">
                     <span class="helper"></span>
                     <a href="https://www.shoppinpal.com/" target="_blank">
-                        <img src="images/users/shoppin-pal.jpg" width="50%"/>
+                        <img src="images/users/shoppin-pal.jpg" width="60%"/>
+                    </a>
+                </div>
+                <div class="col-md-3">
+                    <span class="helper"></span>
+                    <a href="https://www.kumisoft.com/" target="_blank">
```

Comment:
`... target="_blank" rel="noopener noreferrer">`

Likewise, for all other links opening a new window. Why? [Links to cross-origin destinations are unsafe](https://developers.google.com/web/tools/lighthouse/audits/noopener).